Add run `gulp build` clarification

markthethomas opened the pull request:
It seems like this small step was missing in the docs. Doesn't seem to warrant any sort of change in the build steps, but documenting it would be helpful for people trying out the project. Love this project! Really helpful -- thanks!

niftylettuce commented:
@markthethomas Thank you thank you!

niftylettuce commented:
@markthethomas I removed it from Readme.md from your PR (but added a note) and added it to the `postinstall` script in `package.json` where it belongs

markthethomas commented:
Great! Thanks.
On Wed, Jun 10, 2015 at 10:36 AM niftylettuce <notifications@github.com>
wrote:

> @markthethomas <https://github.com/markthethomas> I removed it from
> Readme.md from your PR (but added a note) and added it to the postinstall
> script in package.json where it belongs
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/niftylettuce/gulp-aws-splash/pull/2#issuecomment-110846579>
> .
>
-- 
Mark

---
@markthethomas everywhere
